src/proxy.rs

```diff
@@ -58,13 +58,15 @@ impl<'a> Proxy<'a> {
         P::Error: Into<zbus::Error>,
     {
         let connection = Self::connection().await?;
-        let proxy = zbus::ProxyBuilder::new_bare(&connection)
+        let proxy: zbus::Proxy = zbus::ProxyBuilder::new_bare(&connection)
```

Comment:
```suggestion
        let proxy = zbus::ProxyBuilder::new_bare(&connection)
```
not needed

Reply:
I get 
```
error[E0282]: type annotations needed
  --> src/proxy.rs:64:13
   |
64 |         let inner = zbus::ProxyBuilder::new_bare(&connection)
   |             ^^^^^
...
70 |         let version = inner.get_property::<u32>("version").await.unwrap_or(1);
   |                       ----- type must be known at this point
   |
help: consider giving `inner` an explicit type
   |
64 |         let inner: /* Type */ = zbus::ProxyBuilder::new_bare(&connection)
   |                  ++++++++++++

```
otherwise